apps/builder/app/routes/rest.project.$projectId.tsx

```diff
@@ -22,7 +24,20 @@ export const loader = async ({
       context
     );
 
-    return pagesCanvasData;
+    const project = await projectDb.project.loadByParams(
+      { projectId },
+      context
+    );
+
+    const user =
+      project === null || project.userId === null
+        ? undefined
+        : await getUserById(project.userId);
+
+    return {
+      ...pagesCanvasData,
+      user: user ? { email: user.email } : undefined,
```

Comment:
Not a bug, just for us to keep in mind 
In the only non protected endpoint we are now leaking user email